adapters/adagio/adagio_test.go

```diff
@@ -69,6 +69,7 @@ func TestMakeRequests_Gzip(t *testing.T) {
 		t.Fatalf("Builder returned unexpected error %v", buildErr)
 	}
 
-	requestData, _ := bidder.MakeRequests(&fakeBidRequest, nil)
+	requestData, errors := bidder.MakeRequests(&fakeBidRequest, nil)
```

Comment:
Can you please rename this to `errs` rather? `errors` is the error handling package in the standard Go library and can therefore be confusing

Reply:
@osazos looks like you missed this one

Reply:
you right… done :)